### DISCUSSION TITLE
有Llama2-70B评测结果在Humaneval上的prompt输入和模型输出结果吗？评测没有复现，想借鉴下

### DISCUSSION THREAD

**Discussion by gongjunjin** [open-compass/opencompass#390] [Q&A]
有Llama2-70B评测结果在Humaneval上的prompt输入和模型输出结果吗？评测没有复现，想借鉴下

**Comment by Leymore** [comment]
This is my inference results

[LLaMA-2-70B--openai_humaneval.json.txt](https://github.com/open-compass/opencompass/files/12604478/LLaMA-2-70B--openai_humaneval.json.txt)
